android/src/main/java/com/liferay/mobile/android/auth/CookieSignIn.java

```diff
@@ -266,10 +267,12 @@ protected Call signIn() throws Exception {
 		return client.newCall(builder.build());
 	}
 
-	protected Authenticator authenticator;
+
 	protected CookieManager cookieManager;
 	protected String password;
 	protected Session session;
 	protected String username;
 
+	protected static final Map<String, Authenticator> authenticators =
```

Comment:
Just for understanding, why do we need more than one Authenticator now?

Reply:
In case that the user is using two different backends each one requiring a different authentication